.travis.yml

```diff
@@ -12,6 +13,9 @@ jobs:
     - stage: build-image
       name: "latest"
       script: docker build -f Dockerfile .
+    - stage: build-image
+      name: "postgres-12"
+      script: docker build -f citus_pg_12.dockerfile .
```

Comment:
```suggestion
      script: docker build -f Dockerfile-PG12 .
```

I'd be happier if we had some sort of naming convention in our Dockerfiles. This name is similar to how we named `Dockerfile-alpine` don't you think?

Please rename the file as well, if you decide to do this change.

Reply:
There is a naming convention for default Dockerfiles and it works fine for all IDEs. However, if more than one dockerfile exists, things become a little bit messy. Because IDEs does not understand that file is Dockerfile
But IDEs now understand when using .dockerfile notation
example image as below on VSCode
![image](https://user-images.githubusercontent.com/1596331/110330801-4850b880-802f-11eb-8a19-c66f48e7db1f.png)

Reply:
I prefer changing existing ones after completely grasp the existing devops structure. I don't want to break pipelines now.